src/main/java/net/mcreator/ui/modgui/LootTableGUI.java

```diff
@@ -102,6 +102,24 @@ public LootTableGUI(MCreator mcreator, ModElement modElement, boolean editingMod
 					case "Advancement reward":
 						name.getEditor().setItem("entities/" + currNameNoType);
 						break;
+					case "Archaeology":
+						name.getEditor().setItem("archaeology/" + currNameNoType);
+						break;
+					case "Dispensers":
+						name.getEditor().setItem("dispensers/" + currNameNoType);

```

Comment:
I have checked vanilla code and there is no such folder, applied to some other folders below too.

Please make sure your folders match with what the game expects